packages/@react-spectrum/actionbar/stories/ActionBar.stories.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {action} from '@storybook/addon-actions';
+import {ActionBar, ActionBarContainer, Item} from '../';
+import {Cell, Column, Row, Table, TableBody, TableHeader} from '@react-spectrum/table';
+import React, {useState} from 'react';
+import {Selection} from '@react-types/shared';
+import {SpectrumActionBarProps} from '@react-types/actionbar';
+import {storiesOf} from '@storybook/react';
+
+let columns = [
+  {name: 'Foo', key: 'foo'},
+  {name: 'Bar', key: 'bar'},
+  {name: 'Baz', key: 'baz'}
+];
+
+let items = [
+  {test: 'Test 1', foo: 'Foo 1', bar: 'Bar 1', yay: 'Yay 1', baz: 'Baz 1'},
+  {test: 'Test 2', foo: 'Foo 2', bar: 'Bar 2', yay: 'Yay 2', baz: 'Baz 2'},
+  {test: 'Test 1', foo: 'Foo 3', bar: 'Bar 1', yay: 'Yay 1', baz: 'Baz 1'},
+  {test: 'Test 2', foo: 'Foo 4', bar: 'Bar 2', yay: 'Yay 2', baz: 'Baz 2'},
+  {test: 'Test 1', foo: 'Foo 5', bar: 'Bar 1', yay: 'Yay 1', baz: 'Baz 1'},
+  {test: 'Test 2', foo: 'Foo 6', bar: 'Bar 2', yay: 'Yay 2', baz: 'Baz 2'},
+  {test: 'Test 1', foo: 'Foo 7', bar: 'Bar 1', yay: 'Yay 1', baz: 'Baz 1'},
+  {test: 'Test 2', foo: 'Foo 8', bar: 'Bar 2', yay: 'Yay 2', baz: 'Baz 2'}
+];
+
+storiesOf('ActionBar', module)
+  .add(
+    'default',
+    () => {
+      const [selectedKeys, setSelectedKeys] = useState<Selection>(new Set());
+      return (
+        <ActionBarContainer>
+          <Table
+            selectedKeys={selectedKeys}
+            selectionMode="multiple"
+            onSelectionChange={(keys) => setSelectedKeys(keys)}>
+            <TableHeader columns={columns}>
+              {column => <Column>{column.name}</Column>}
+            </TableHeader>
+            <TableBody items={items}>
+              {item =>
+                (<Row key={item.foo}>
+                  {key => <Cell>{item[key]}</Cell>}
+                </Row>)
+              }
+            </TableBody>
+          </Table>
+          <ActionBar
+            selectedItemCount={selectedKeys === 'all' ? selectedKeys : selectedKeys.size}
```

Comment:
your types say that selectedItemCount is a number, yet you pass it a string if it's 'all'
I don't see where you handle 'all' anywhere else